### DISCUSSION TITLE
Search with special characters

### DISCUSSION THREAD

**Discussion by mikkvitchko** [squidfunk/mkdocs-material#2562] [Q&A] [answered]
Not sure if it is relevant, please redirect me if not .

Search doesn't work for terms starting from "@".

For example search for "support@" - works (finds "support@where.be"), but search for "@PATH" - doesn't (search for "PATH" finds pages with "@PATH" just fine).

Is there a way to correct this behavior?

**Comment by squidfunk** [comment, marked as answer]
The search is implemented with Lunr.js, and theoretically, you could add a custom pipeline to expand the whole email address as described in [their docs](https://lunrjs.com/guides/core_concepts.html#text-processing). You could fork Material for MkDocs to add custom pipeline functions.

**Reply by mikkvitchko** [reply]
Thank you, @squidfunk
